iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -414,7 +416,7 @@ public boolean setColStatistics(org.apache.hadoop.hive.ql.metadata.Table hmsTabl
       List<ColumnStatistics> colStats) {
     Table tbl = IcebergTableUtil.getTable(conf, hmsTable.getTTable());
     String snapshotId = String.format("%s-STATS-%d", tbl.name(), tbl.currentSnapshot().snapshotId());
-    invalidateStats(getStatsPath(tbl));
+    checkAndMergeOrInvalidateStats(colStats.get(0), hmsTable);
```

Comment:
would it be always size 1? what it it's empty?

Reply:
For now, it will be size one implying all the column stats are stored as a single blob. 
But in the next PR when we implement retrieving stats for particular columns this will be modified to store multiple blobs.